src/App.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import logo from "./logo.svg";
+import Logo from "./logo.svg";
```

Comment:
Is there a naming scheme for images?  Caps or lower case?

Reply:
Good question. I think I like the uppercase better.